server_addon/maya/server/settings/publishers.py

```diff
@@ -38,26 +38,26 @@ def angular_unit_enum():
 def extract_alembic_flags_enum():
     """Get flags for alembic extraction enumerator."""
     return [
-        {"label": "autoSubd", "value": "autoSubd"},
+        {"value": "autoSubd", "label": "Auto Subd"},
         {
-            "label": "dontSkipUnwrittenFrames",
-            "value": "dontSkipUnwrittenFrames"
+            "value": "dontSkipUnwrittenFrames",
+            "label": "Dont Skip Unwritten Frames"
         },
-        {"label": "eulerFilter", "value": "eulerFilter"},
-        {"label": "noNormals", "value": "noNormals"},
-        {"label": "preRoll", "value": "preRoll"},
-        {"label": "renderableOnly", "value": "renderableOnly"},
-        {"label": "stripNamespaces", "value": "stripNamespaces"},
-        {"label": "uvWrite", "value": "uvWrite"},
-        {"label": "uvsOnly", "value": "uvsOnly"},
-        {"label": "verbose", "value": "verbose"},
-        {"label": "wholeFrameGeo", "value": "wholeFrameGeo"},
-        {"label": "worldSpace", "value": "worldSpace"},
-        {"label": "writeColorSets", "value": "writeColorSets"},
-        {"label": "writeFaceSets", "value": "writeFaceSets"},
-        {"label": "writeNormals", "value": "writeNormals"},
-        {"label": "writeUVSets", "value": "writeUVSets"},
-        {"label": "writeVisibility", "value": "writeVisibility"}
+        {"value": "eulerFilter", "label": "Euler Filter"},
+        {"value": "noNormals", "label": "No Normals"},
+        {"value": "preRoll", "label": "Pre Roll"},
+        {"value": "renderableOnly", "label": "Renderable Only"},
+        {"value": "stripNamespaces", "label": "Strip Namespaces"},
+        {"value": "uvWrite", "label": "UV Write"},
+        {"value": "uvsOnly", "label": "UVs Only"},
+        {"value": "verbose", "label": "Verbose"},
+        {"value": "wholeFrameGeo", "label": "Whole Frame Geo"},
+        {"value": "worldSpace", "label": "World Space"},
+        {"value": "writeColorSets", "label": "Write Color Sets"},
+        {"value": "writeFaceSets", "label": "Write Face Sets"},
+        {"value": "writeNormals", "label": "Write Normals"},
+        {"value": "writeUVSets", "label": "Write UV Sets"},
+        {"value": "writeVisibility", "label": "Write Visibility"}
```

Comment:
I may have OCD - but can we align both of the `enum()` methods to either for both start the entries with the `value` or the `label`. They now seem the opposite. It's cosmetics, but makes it easier to parse if things are consistent throughout the file.